commit befa8cec4950e7aa71c407252e1bbcf11b512f39
Author: Janez Troha <dz0ny@ubuntu.si>
Date:   Fri Feb 6 22:45:25 2015 +0100

    Add Ubuntu Snappy support

diff --git a/include/configs/mx6_cubox-i.h b/include/configs/mx6_cubox-i.h
index 51532be..ba2144a 100644
--- a/include/configs/mx6_cubox-i.h
+++ b/include/configs/mx6_cubox-i.h
@@ -315,6 +315,7 @@
 /* Miscellaneous configurable options */
 #define CONFIG_SYS_LONGHELP
 #define CONFIG_SYS_HUSH_PARSER
+#define CONFIG_SUPPORT_RAW_INITRD
 
 #ifndef __ASSEMBLY__
 extern char *config_sys_prompt;
